dashboard-for-web/articles/web-dashboard-designer-mode/ui-elements/dialogs-and-wizards/query-builder.md

```diff
@@ -67,9 +67,9 @@ The following settings are available for each column.
 ## Filter Data
```

Comment:
Line 58: The **Output** field allows you to choose whether to include the selected column to the query.

Can you change "to the query" to "in the query"?